### PR TITLE
GoFr Updated Readme with better readability. 

### PR DESCRIPTION
**Pull request by abhirajadhikary06**
**Description:**

- This pull request addresses issue #1131 by updating the README.md file with the following improvements:

1. Centered the "GoFr" heading and logo for a more visually appealing introduction.
2. Updated the badges to a more modern version for better alignment with contemporary styles.
3. Added emojis and highlighted key sections to enhance the visual structure.

- These changes aim to improve the visual aesthetics and readability of the README, making it more professional and user-friendly.

**Breaking Changes (if applicable):**

-  No breaking changes are introduced in this pull request.

**Additional Information:**

- No new dependencies or external libraries have been used in this update.

- Below is a screenshot of the updated layout:
image
image
image
image

**Checklist:**

-   [x] I have formatted my code using  `goimport`  and  `golangci-lint`.
-   [x] This PR does not decrease the overall code coverage.
-   [x] All new code is covered by unit tests.
-   [x] I have reviewed the code comments and documentation for clarity.

### CODE REVIEW

**Comment by abhirajadhikary06** [comment]
@Umang01-hash I have made a PR please check it out.

**Comment by coolwednesday** [comment]
@abhirajadhikary06 Please give a better name to the PR, that describes the solution and not add your name.

**Comment by abhirajadhikary06** [comment]
Ok, thanks for informing

File: README.md [hunk 1]
```diff
@@ -67,35 +78,46 @@ func main() {
...
-If you want to say thank you and/or support the active development of GoFr:
+---
 
-1. [Star](https://docs.github.com/en/get-started/exploring-projects-on-github/saving-repositories-with-stars) the repo.
-2. Write a review or tutorial on [Medium](https://medium.com/), [Dev.to](https://dev.to/) or personal blog.
-3. Visit [CONTRIBUTING](CONTRIBUTING.md) for details on submitting patches and the contribution workflow.
+### 🎁 **Get a GoFr T-Shirt & Stickers!**
 
-If your PR is merged or you have written an article or contributed in someway to development or spreading the word about GoFr, fill the [Google Form](https://forms.gle/R1Yz7ZzY3U5WWTgy5), and we will send you a <b>GoFr T-Shirt and Stickers</b> as a token of appreciation.
+If your PR gets merged or you contribute through articles or spreading the word about GoFr, fill out [this form](https://forms.gle/R1Yz7ZzY3U5WWTgy5) to claim your **GoFr T-shirt** and stickers as a thank-you gift!
```
Comment (coolwednesday) [inline README.md:123]: I feel you can write it like this,

If your PR is merged, or if you contribute by writing articles or promoting GoFr, we invite you to fill out [this form](https://forms.gle/R1Yz7ZzY3U5WWTgy5) to claim your **GoFr merchandise** as a token of our appreciation!

**Review by coolwednesday — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (abhirajadhikary06) [reply README.md:123]: Sure.

Comment (abhirajadhikary06) [reply README.md]: @coolwednesday The update provided by you is resolved.

**Review by abhirajadhikary06 — COMMENTED** [review]
Thanks for proper updates.

**Review by Umang01-hash — APPROVED** [review]

**Comment by abhirajadhikary06** [comment]
Could any admin please merge my PR with the main branch.

**Pull request merged**